index.js

```diff
@@ -81,13 +81,15 @@ async function underPressure (fastify, opts) {
   }
 
   fastify.decorate('memoryUsage', memoryUsage)
+  fastify.decorate('isUnderPressureError', false)
```

Comment:
Is this a private module status?
If so we could use a `Symbol` for this kind of usage